<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<meta name="viewport"    content="width=device-width, initial-scale=1.0">
	<meta name="description" content="Thibaut Jombart's webpage">
	<meta name="author"      content="Thibaut Jombart">
	<meta name="keywords" content="Thibaut,Jombart,RECON,outbreaks,repidemicsconsortium,official,webpage,website,adegenet">
	
	<title>Publications</title>

	<link rel="shortcut icon" href="https://thibautjombart.netlify.app/images/gt_favicon.png">

	
	<link href="https://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.no-icons.min.css" rel="stylesheet">
	
	<link href="https://thibautjombart.netlify.app/css/bootstrap.no-icons.min.css" rel="stylesheet">
	
	
	
	<link href="https://thibautjombart.netlify.app/css/font-awesome.css" rel="stylesheet">
	
	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Alice|Open+Sans:400,300,700">
	
	<link rel="stylesheet" href="https://thibautjombart.netlify.app/css/styles.css">

	

    
        <script type="text/javascript" src="//platform-api.sharethis.com/js/sharethis.js#property=&product=inline-share-buttons"></script>
    

</head>
<body class="home">

<header id="header">
	<div id="head" class="parallax" parallax-speed="0">
		<h1 id="logo" class="text-center">
			<img class="img-circle" src="https://thibautjombart.netlify.app/images/thibaut-jombart.jpg" alt="">
			<span class="title">Thibaut Jombart</span>
			
			<span class="tagline">Senior Lecturer in Outbreak Analytics<br>
			  <a href="">thibautjombart@gmail.com</a>
			</span>
		</h1>
	</div>

    <nav class="navbar navbar-default navbar-sticky">
    <div class="container-fluid">

        <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="true">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
        </div>

        <div class="navbar-collapse collapse" id="bs-example-navbar-collapse-1">

            <ul class="nav navbar-nav">
                
                <li>
                    <a href="/">home</a>
                </li>
                
                <li>
                    <a href="/about/">About</a>
                </li>
                
                <li>
                    <a href="/group/">Group</a>
                </li>
                
                <li>
                    <a href="/post/">blog</a>
                </li>
                
                <li>
                    <a href="/resources/">Resources</a>
                </li>
                
                <li>
                    <a href="/publications/">Publications</a>
                </li>
                
                <li>
                    <a href="/contact/">Contact</a>
                </li>
                
                
            </ul>

        </div>
    </div>
</nav>


</header>


<main id="main">

	<div class="container">

		<div class="row topspace">

			
			<aside class="col-sm-4 sidebar sidebar-right">

				<div class="nav text-right nav-side">
					<li class="active"><a href="#">Publications</a></li>
				</div>

			</aside>
			

			
			<article class="col-sm-8 maincontent">
                <h3 id="statistics">Statistics</h3>
<p>These were last updated on the 01-03-2024.</p>
<p>Web of science:</p>
<ul>
<li>citations: 15,570</li>
<li>h-index: 46</li>
</ul>
<p>Google scholar:</p>
<ul>
<li>citations: 40,144</li>
<li>h-index: 65</li>
<li>h10-index: 108</li>
</ul>
<h3 id="list-of-publications">List of publications</h3>
<p>This list was last updated on the 01-01-2024. It does not include the (many)
papers on COVID-19 under the CMMID group authorship.</p>
<ul>
<li>
<p>Evans B, Keiser O, Kaiser L, <strong>Jombart T</strong> (2023) Analysis of global routine immunisation coverage shows disruption and stagnation during the first two-years of the COVID-19 pandemic with tentative recovery in 2022. <em>Vaccine X</em> 100383</p>
</li>
<li>
<p>Qian GY, Edmunds JW, Bausch DG, <strong>Jombart T</strong> (2023) A mathematical model of Marburg virus disease outbreaks and the potential role of vaccination in control. <em>BMC Medicine</em> 21(1):439.</p>
</li>
<li>
<p>Geismar C, Nguyen V, Fragaszy E, Shrotri M, Navaratnam AMD, Beale S, Byrne TE, Fong  WLE, Yavlinsky A, Kovar J, Hoskins S, Braithwaite I, Aldridge RW, Hayward AC, White PJ, <strong>Jombart T</strong>, Cori A (2023) Bayesian reconstruction of SARS-CoV-2 transmissions highlights substantial proportion of negative serial intervals. <em>Epidemics</em> 100713</p>
</li>
<li>
<p>Zeng W, Mobula L, Samaha H, Yao M, Ahuka S, Wilkinson T, <strong>Jombart T</strong> et al. (2023) Cost of public health interventions to respond to the 10th Ebola outbreak in the Democratic Republic of the Congo. <em>BMJ Global Health</em> 8(10):e012660</p>
</li>
<li>
<p>Qian GY, <strong>Jombart T</strong>, Edmunds JW (2023) Assessing the feasibility of Phase 3 vaccine trials against Marburg Virus Disease: A modelling study. <em>Vaccine X</em> 14:100321.</p>
</li>
<li>
<p>Evans B, <strong>Jombart T</strong> (2022) Worldwide routine immunisation coverage regressed during the first year of the COVID-19 pandemic. <em>Vaccine</em> 40(26):3531-3535. doi: 10.1016/j.vaccine.2022.01.044</p>
</li>
<li>
<p><strong>Jombart T</strong> (2021) Why development of outbreak analytics tools should be valued, supported, and funded. <em>Lancet ID</em> 21:4, 458-459.</p>
</li>
<li>
<p>Abbas M, Robalo Nunes T, Cori A, Cordey S, Laubscher F, Baggio S, <strong>Jombart T</strong>, et al. (2021) Explosive nosocomial outbreak of SARS-CoV-2 in a rehabilitation clinic: the limits of genomics for outbreak reconstruction. <em>J Hosp Infect</em> 117:124-134</p>
</li>
<li>
<p>Brooks-Pollock E, Danon L, <strong>Jombart T</strong>, Pellis L. (2021) Modelling that shaped the early COVID-19 pandemic response in the UK. <em>Philos Trans R Soc Lond B Biol Sci</em>. 2021;376: 20210001. <a href="https://royalsocietypublishing.org/doi/10.1098/rstb.2021.0001"><a href="https://doi.org/10.1098/rstb.2021.0001">https://doi.org/10.1098/rstb.2021.0001</a></a></p>
</li>
<li>
<p><strong>Jombart T</strong>, Ghozzi S, Schumacher D, Taylor TJ, Leclerc QJ, Jit M, et al. (2021) Real-time monitoring of COVID-19 dynamics using automated trend fitting and anomaly detection. <em>Philos Trans R Soc Lond B Biol Sci</em>. 2021;376: 20200266.<a href="https://royalsocietypublishing.org/doi/10.1098/rstb.2020.0266"><a href="https://doi.org/10.1098/rstb.2020.0266">https://doi.org/10.1098/rstb.2020.0266</a></a></p>
</li>
<li>
<p>Campbell F, Archer B, Laurenson-Schafer H, Jinnai Y, Konings F, Batra N, Pavlin B, Vandemale K, Van Kerkhove M, <strong>Jombart T</strong> et al. (2021) Increased transmissibility and global spread of SARS-CoV-2 variants of concern as at June 2021. <em>Euro Surveill</em>. 2021;26. <a href="https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2021.26.24.2100509"><a href="https://doi.org/10.2807/1560-7917.ES.2021.26.24.2100509">https://doi.org/10.2807/1560-7917.ES.2021.26.24.2100509</a></a></p>
</li>
<li>
<p>Leclerc QJ, Nightingale ES, Abbott S, CMMID COVID-19 Working Group, <strong>Jombart T</strong> (2021) Analysis of temporal trends in potential COVID-19 cases reported through NHS Pathways England. <em>Scientific Reports</em>. 2021;11: 7106. <a href="https://www.nature.com/articles/s41598-021-86266-3"><a href="https://doi.org/10.1038/s41598-021-86266-3">https://doi.org/10.1038/s41598-021-86266-3</a></a></p>
</li>
<li>
<p>Carter SE, Gobat N, Pfaffmann Zambruni J, Bedford J, van Kleef E, <strong>Jombart T</strong>, et al. (2021)
What questions we should be asking about COVID-19 in humanitarian settings: perspectives from the Social Sciences Analysis Cell in the Democratic Republic of the Congo. <em>BMJ Global Health</em> 2020;5:e003607. <a href="https://gh.bmj.com/content/5/9/e003607">10.1136/bmjgh-2020-003607</a></p>
</li>
<li>
<p>Rees EM, Nightingale ES, Jafari Y, Waterlow NR, Clifford S, B Pearson CA, <strong>Jombart T</strong> et al. (2020) COVID-19 length of hospital stay: a systematic review and data synthesis. BMC Med. 2020;18: 270.<a href="https://doi.org/10.1186/s12916-020-01726-3"><a href="https://doi.org/10.1186/s12916-020-01726-3">https://doi.org/10.1186/s12916-020-01726-3</a></a></p>
</li>
<li>
<p>Mobula LM, Samaha H, Yao M, Gueye AS, Diallo B, Umutoni C, Anoko J, Lokonga JP, Minikulu L, Mossoko M, Bruni E, Carter S, <strong>Jombart T</strong>, et al. (2020) Recommendations for the COVID-19 Response at the National Level Based on Lessons Learned from the Ebola Virus Disease Outbreak in the Democratic Republic of the Congo. <em>The American Journal of Tropical Medicine and Hygiene</em>: 12â€“17. doi:10.4269/ajtmh.20-0256</p>
</li>
<li>
<p>Parisi A, Phuong TLT, Mather AE, <strong>Jombart T</strong>, Tuyen HT, Lan NPH, et al. (2020) The role of animals as a source of antimicrobial resistant nontyphoidal Salmonella causing invasive and non-invasive human disease in Vietnam. <em>Infect Genet Evol.</em> 2020;85: 104534.</p>
</li>
<li>
<p>Jit M, <strong>Jombart T</strong>, Nightingale ES, et al. 2020. Estimating number of cases and spread of coronavirus disease (COVID-19) using critical care admissions, United Kingdom, February to March 2020. Euro Surveill. 25(18):2000632. <a href="https://doi.org/10.2807/1560-7917.ES.2020.25.18.2000632">doi:10.2807/1560-7917.ES.2020.25.18.2000632</a></p>
</li>
<li>
<p><strong>Jombart T</strong>, van Zandvoort K, Russell TW et al. Inferring the number of COVID-19 cases from recently reported deaths. Wellcome Open Res 2020, 5:78 <a href="https://doi.org/10.12688/wellcomeopenres.15786.1">10.12688/wellcomeopenres.15786.1</a></p>
</li>
<li>
<p><strong>Jombart T</strong>, Jarvis CI, Mesfin S, Tabal N, Mossoko M, Mpia LM, Abedi A, Chene S, Forbin EE, Belizaire MRD, de RadiguÃ¨s X, Ngombo R, Tutu Y, Finger F, Crowe M, Edmunds WJ, Nsio J, Yam A, Diallo B, Gueye AS, Ahuka-Mundeke S, Yao M, Fall IS (2020) The cost of insecurity: from flare-up to control of a major Ebola virus disease hotspot during the outbreak in the Democratic Republic of the Congo, 2019. Eurosurveillance 25(2). <a href="https://doi.org/10.2807/1560-7917.ES.2020.25.2.1900735">10.2807/1560-7917.ES.2020.25.2.1900735</a></p>
</li>
<li>
<p>Leiva, C. et al. Population substructure and signals of divergent adaptive selection despite admixture in the sponge Dendrilla antarctica from shallow waters surrounding the Antarctic Peninsula. <i>Molecular Ecology</i> (2019). DOI: <a href="https://doi.org/10.1111/mec.15135">10.1111/mec.15135</a></p>
</li>
<li>
<p>Campbell F, Cori A, Ferguson N, <strong>Jombart T</strong> (2019) Bayesian inference of transmission chains using timing of symptoms, pathogen genomes and contact data. <i>PLoS Comput Biol</i> 15(3): e1006930. DOI:<a href="https://doi.org/10.1371/journal.pcbi.1006930">10.1371/journal.pcbi.1006930</a></p>
</li>
<li>
<p>Dighe, A., <strong>Jombart, T.</strong>, Van Kerkhove, M. D. &amp; Ferguson, N. (2019) A systematic review of MERS-CoV seroprevalence and RNA prevalence in dromedary camels: Implications for animal vaccination
<i>Epidemics</i> 100350. DOI: <a href="https://doi.org/10.1016/j.epidem.2019.100350">10.1016/j.epidem.2019.100350</a></p>
</li>
<li>
<p>Thomas R. Sewell, Jianing Zhu, Johanna Rhodes, Ferry Hagen, Jacques F. Meis, Matthew C. Fisher, <strong>Thibaut Jombart</strong>
Nonrandom Distribution of Azole Resistance across the Global Population of Aspergillus fumigatus (2019)
<i>mBio</i> 10 (3) e00392-19; DOI: <a href="https://doi.org/10.1128/mBio.00392-19">10.1128/mBio.00392-19</a></p>
</li>
<li>
<p>Kamvar ZN, Cai J, Pulliam JRC, Schumacher J &amp; <strong>Jombart T</strong> (2019) Epidemic curves made easy using the R package incidence. <i>F1000Res</i> 8: 139.</p>
</li>
<li>
<p>Polonsky JA, Baidjoe A, Kamvar ZN, Cori A, Durski K, Edmunds JW, Eggo RM, Funk S, Kaiser L, Keating P, le Polain de Waroux O, Marks M, Moraga P, Morgan O, Nouvellet P, Ratnayake R, Roberts CH, Whitworth J &amp; <strong>Jombart T</strong>
(2019). Outbreak analytics: a developing data science for informing the response to emerging pathogens <i>Philosophical Transactions B</i> 374 (1776). DOI: <a href="https://doi.org/10.1098/rstb.2018.0276">10.1098/rstb.2018.0276</a></p>
</li>
<li>
<p>Cori A, Nouvellet P, Garske T, Bourhy H, NakounÃ© E &amp; <strong>Jombart T</strong>
(2018). A graph-based evidence synthesis approach to detecting
outbreak clusters: an application to dog rabies <i>PLoS Computational
Biology</i> DOI: <a href="https://doi.org/10.1371/journal.pcbi.1006554">10.1371/journal.pcbi.1006554</a></p>
</li>
<li>
<p>Campbell F, Didelot X, FitzJohn R, Ferguson N, Cori A &amp; <strong>Jombart T</strong>
(2018). <i>outbreaker2</i>: a modular platform for outbreak
reconstruction <i>BMC Bioinformatics</i>
doi:<a href="https://doi.org/10.1186/s12859-018-2330-z">10.1186/s12859-018-2330-z</a></p>
</li>
<li>
<p>Campbell, F., Strang, C., Ferguson, N., Cori, A. &amp; <strong>Jombart, T.</strong>
(2018). When are pathogen genome sequences informative of
transmission events? <i>PLoS Pathogens</i>
doi:<a href="http://doi.org/10.1186/s12859-017-1988-y">10.1371/journal.ppat.1006885</a></p>
</li>
<li>
<p>Montano, V., &amp; <strong>Jombart, T.</strong> (2017). An Eigenvalue test for spatial
principal component analysis. <i>BMC Bioinformatics</i>,
<i>18</i>(1), 562. doi:<a
href="http://doi.org/10.1186/s12859-017-1988-y">10.1186/s12859-017-1988-y</a></p>
</li>
<li>
<p>Paradis, E., Gosselin, T., Grunwald, N. J., <strong>Jombart, T.</strong>, Manel, S., &amp;
Lapp, H. (2017). Towards an integrated ecosystem of R packages for
the analysis of population genetic data. <i>Molecular Ecology
Resources</i>, <i>17</i>(1), 1-4. doi:<a href="http://doi.org/10.1111/1755-0998.12636">10.1111/1755-0998.12636</a></p>
</li>
<li>
<p>Dupuis, J. R., Bremer, F. T., <strong>Jombart, T.</strong>, Sim, S. B., &amp; Geib, S.
M. (2017). mvmapper: Interactive spatial mapping of genetic
structures.. <i>Mol Ecol Resour</i>. doi:<a href="http://doi.org/10.1111/1755-0998.12724">10.1111/1755-0998.12724</a></p>
</li>
<li>
<p><strong>Jombart, T.</strong>, Kendall, M., Almagro-Garcia, J., &amp; Colijn, C.
(2017). treespace: Statistical exploration of landscapes of
phylogenetic trees. <i>Molecular Ecology Resources</i>, <i>17</i>(6),
1385-1392. doi:<a href="http://doi.org/10.1111/1755-0998.12676">10.1111/1755-0998.12676</a></p>
</li>
<li>
<p>Cori, A., Donnelly, C. A., Dorigatti, I., Ferguson, N. M., Fraser,
C., Garske, T., &hellip; Blake, I. M. (2017). Key data for outbreak
evaluation: building on the Ebola experience. <i>Philosophical
Transactions of the Royal Society B-Biological Sciences</i>,
<i>372</i>(1721), 14 pages. doi:<a href="http://doi.org/10.1098/rstb.2016.0371">10.1098/rstb.2016.0371</a></p>
</li>
<li>
<p>Garske, T., Cori, A., Ariyarajah, A., Blake, I. M., Dorigatti, I.,
Eckmanns, T., &hellip; Donnelly, C. A. (2017). Heterogeneities in the case
fatality ratio in the West African Ebola outbreak 2013-2016.
<i>Philosophical Transactions of the Royal Society B-Biological
Sciences</i>, <i>372</i>(1721), 9 pages. doi:<a href="http://doi.org/10.1098/rstb.2016.0308">10.1098/rstb.2016.0308</a></p>
</li>
<li>
<p>Nouvellet, P., Cori, A., Garske, T., Blake, I. M., Dorigatti, I.,
Hinsley, W., &hellip; Riley, S. (2017). A simple approach to measure
transmissibility and forecast incidence.. <i>Epidemics</i>.
doi:<a href="http://doi.org/10.1016/j.epidem.2017.02.012">10.1016/j.epidem.2017.02.012</a></p>
</li>
<li>
<p>Bertranpetit, E., <strong>Jombart, T.</strong>, Paradis, E., Pena, H., Dubey, J., Su,
C., &hellip; Ajzenberg, D. (2017). Phylogeography of Toxoplasma gondii
points to a South American origin. <i>Infection Genetics and
Evolution</i>, <i>48</i>, 150-155. doi:<a href="http://doi.org/10.1016/j.meegid.2016.12.020">10.1016/j.meegid.2016.12.020</a></p>
</li>
<li>
<p>Agua-Agum, J., Ariyarajah, A., Aylward, B., Bawo, L., Bilivogui, P.,
Blake, I. M., &hellip; Yoti, Z. (2016). Exposure Patterns Driving Ebola
Transmission in West Africa: A Retrospective Observational Study.
<i>PLoS Medicine</i>, <i>13</i>(11), 23 pages.
doi:<a href="http://doi.org/10.1371/journal.pmed.1002170">10.1371/journal.pmed.1002170</a></p>
</li>
<li>
<p>Inns, T., Ashton, P. M., Herrera-Leon, S., Lighthill, J., Foulkes,
S., <strong>Jombart, T.</strong>, &hellip; Vivancos, R. (2017). Prospective use of whole
genome sequencing (WGS) detected a multi-country outbreak of
Salmonella Enteritidis. <i>EPIDEMIOLOGY AND INFECTION</i>, <i>145</i>(2),
289-298. doi:<a href="http://doi.org/10.1017/S0950268816001941">10.1017/S0950268816001941</a></p>
</li>
<li>
<p>Clare, F. C., Halder, J. B., Daniel, O., Bielby, J., Semenov, M. A.,
<strong>Jombart, T.</strong>, &hellip; Fisher, M. C. (2016). Climate forcing of an emerging
pathogenic fungus across a montane multi-host community.
<i>Philosophical Transactions of the Royal Society B-Biological
Sciences</i>, <i>371</i>(1709), 8 pages. doi:<a href="http://doi.org/10.1098/rstb.2015.0454">10.1098/rstb.2015.0454</a></p>
</li>
<li>
<p>Paradis, E., Gosselin, T., Goudet, J., <strong>Jombart, T.</strong>, &amp; Schliep, K.
(2017). Linking genomics and population genetics with R. <i>Molecular
Ecology Resources</i>, <i>17</i>(1), 54-66.
doi:<a href="http://doi.org/10.1111/1755-0998.12577">10.1111/1755-0998.12577</a></p>
</li>
<li>
<p>Dallman, T., Inns, T., <strong>Jombart, T.</strong>, Ashton, P., Loman, N., Chatt, C.,
&hellip; Cleary, P. (2016). Phylogenetic structure of European Salmonella
Enteritidis outbreak correlates with national and international egg
distribution network.. <i>Microb Genom</i>, <i>2</i>(8), e000070.
doi:<a href="http://doi.org/10.1099/mgen.0.000070">10.1099/mgen.0.000070</a></p>
</li>
<li>
<p><strong>Jombart, T.</strong>, Archer, F., Schliep, K., Kamvar, Z., Harris, R.,
Paradis, E., &hellip; Lapp, H. (2017). apex: phylogenetics with multiple
genes. <i>MOLECULAR ECOLOGY RESOURCES</i>, <i>17</i>(1), 19-26.
doi:<a href="http://doi.org/10.1111/1755-0998.12567">10.1111/1755-0998.12567</a></p>
</li>
<li>
<p>Agua-Agum, J., Allegranzi, B., Ariyarajah, A., Aylward, R. B., Blake,
I. M., Barboza, P., &hellip; Van Kerkhove, M. D. (2016). After Ebola in
West Africa - Unpredictable Risks, Preventable Epidemics. <i>NEW
ENGLAND JOURNAL OF MEDICINE</i>, <i>375</i>(6), 587-596.
doi:<a href="http://doi.org/10.1056/NEJMsr1513109">10.1056/NEJMsr1513109</a></p>
</li>
<li>
<p>Cauchemez, S., Nouvellet, P., Cori, A., <strong>Jombart, T.</strong>, Garske, T.,
Clapham, H., &hellip; Ferguson, N. M. (2016). Unraveling the drivers of
MERS-CoV transmission. <i>Proceedings of the National Academy of
Sciences of the United States of America</i>, <i>113</i>(32),
9081-9086. doi:<a href="http://doi.org/10.1073/pnas.1519235113">10.1073/pnas.1519235113</a></p>
</li>
<li>
<p>Lessler, J., Salje, H., Van Kerkhove, M. D., Ferguson, N. M.,
Cauchemez, S., Rodriquez-Barraquer, I., &hellip; Cummings, D. A. T.
(2016). Estimating the Severity and Subclinical Burden of Middle East
Respiratory Syndrome Coronavirus Infection in the Kingdom of Saudi
Arabia. <i>American Journal of Epidemiology</i>, <i>183</i>(7),
657-663. doi:<a href="http://doi.org/10.1093/aje/kwv452">10.1093/aje/kwv452</a></p>
</li>
<li>
<p>Agua-Agum, J., Ariyarajah, A., Blake, I. M., Cori, A., Donnelly, C.
A., Dorigatti, I., &hellip; Van Kerkhove, M. D. (2016). Ebola Virus
Disease among Male and Female Persons in West Africa. <i>New England
Journal of Medicine</i>, <i>374</i>(1), 96-98.
doi:<a href="http://doi.org/10.1056/NEJMc1510305">10.1056/NEJMc1510305</a></p>
</li>
<li>
<p>Karkey, A., <strong>Jombart, T.</strong>, Walker, A. W., Thompson, C. N., Torres, A.,
Dongol, S., &hellip; Baker, S. (2016). The Ecological Dynamics of Fecal
Contamination and Salmonella Typhi and Salmonella Paratyphi A in
Municipal Kathmandu Drinking Water. <i>PLoS Neglected Tropical
Diseases</i>, <i>10</i>(1), 18 pages.
doi:<a href="http://doi.org/10.1371/journal.pntd.0004346">10.1371/journal.pntd.0004346</a></p>
</li>
<li>
<p>Finnie, T. J. R., South, A., Bento, A., Sherrard-Smith, E., &amp;
<strong>Jombart, T.</strong> (2016). EpiJSON: A unified data-format for epidemiology.
<i>EPIDEMICS</i>, <i>15</i>, 20-26. doi:<a href="http://doi.org/10.1016/j.epidem.2015.12.002">10.1016/j.epidem.2015.12.002</a></p>
</li>
<li>
<p>Nouvellet, P., Garske, T., Mills, H. L., Nedjati-Gilani, G., Hinsley,
W., Blake, I. M., &hellip; Ferguson, N. M. (2015). The role of rapid
diagnostics in managing Ebola epidemics. <i>Nature</i>, <i>528</i>(7580),
S109-S116. doi:<a href="http://doi.org/10.1038/nature16041">10.1038/nature16041</a></p>
</li>
<li>
<p><strong>Jombart, T.</strong>, Kendall, M. L., Almagro-Garcia, J., &amp; Colijn, C.
(2015). treespace (Version 1.0.0) [Computer Software].</p>
</li>
<li>
<p>Weinert, L. A., Chaudhuri, R. R., Wang, J., Peters, S. E., Corander,
J., <strong>Jombart, T.</strong>, &hellip; Maskell, D. J. (2015). Genomic signatures of
human and animal disease in the zoonotic pathogen Streptococcus suis
(vol 6, 6740, 2015). <i>NATURE COMMUNICATIONS</i>, <i>6</i>, 1 page.
doi:<a href="http://doi.org/10.1038/ncomms8272">10.1038/ncomms8272</a></p>
</li>
<li>
<p>Weinert, L. A., Chaudhuri, R. R., Wang, J., Peters, S. E., Corander,
J., <strong>Jombart, T.</strong>, &hellip; Maskell, D. J. (2015). Genomic signatures of
human and animal disease in the zoonotic pathogen Streptococcus suis.
<i>NATURE COMMUNICATIONS</i>, <i>6</i>, 9 pages.
doi:<a href="http://doi.org/10.1038/ncomms7740">10.1038/ncomms7740</a></p>
</li>
<li>
<p>Agua-Agum, J., Ariyarajah, A., Blake, I. M., Cori, A., Donnelly, C.
A., Dorigatti, I., &hellip; Van Kerkhove, M. D. (2015). Ebola Virus
Disease among Children in West Africa. <i>NEW ENGLAND JOURNAL OF
MEDICINE</i>, <i>372</i>(13), 1274-1277. doi:<a href="http://doi.org/10.1056/NEJMc1415318">10.1056/NEJMc1415318</a></p>
</li>
<li>
<p>Hao, C. T., Karkey, A., Duy, P. T., Boinett, C. J., Cain, A. K.,
Ellington, M., &hellip; Baker, S. (2015). A high-resolution genomic
analysis of multidrug-resistant hospital outbreaks of Klebsiella
pneumoniae. <i>EMBO Molecular Medicine</i>, <i>7</i>(3), 227-239.</p>
</li>
<li>
<p>West African Ebola Epidemic after One Year - Slowing but Not Yet
under Control (2015). <i>New England Journal of Medicine</i>, <i>372</i>(6),
584-587. doi:<a href="http://doi.org/10.1056/NEJMc1414992">10.1056/NEJMc1414992</a></p>
</li>
<li>
<p>Tong, S. Y. C., Holden, M. T. G., Nickerson, E. K., Cooper, B. S.,
Koeser, C. U., Cori, A., &hellip; Peacock, S. J. (2015). Genome sequencing
defines phylogeny and spread of methicillin-resistant Staphylococcus
aureus in a high transmission setting. <i>GENOME RESEARCH</i>, <i>25</i>(1),
111-118. doi:<a href="http://doi.org/10.1101/gr.174730.114">10.1101/gr.174730.114</a></p>
</li>
<li>
<p>Nhung, N. T., Cuong, N. V., Campbell, J., Hoa, N. T., Bryant, J. E.,
Truc, V. N. T., &hellip; Carrique-Mas, J. (2015). High Levels of
Antimicrobial Resistance among Escherichia coli Isolates from
Livestock Farms and Synanthropic Rats and Shrews in the Mekong Delta
of Vietnam. <i>APPLIED AND ENVIRONMENTAL MICROBIOLOGY</i>, <i>81</i>(3),
812-820. doi:<a href="http://doi.org/10.1128/AEM.03366-14">10.1128/AEM.03366-14</a></p>
</li>
<li>
<p>Aylward, B., Barboza, P., Bawo, L., Bertherat, E., Bilivogui, P.,
Blake, I., &hellip; Yoti, Z. (2014). Ebola Virus Disease in West Africa -
The First 9 Months of the Epidemic and Forward Projections. <i>NEW
ENGLAND JOURNAL OF MEDICINE</i>, <i>371</i>(16), 1481-1495.
doi:<a href="http://doi.org/10.1056/NEJMoa1411100">10.1056/NEJMoa1411100</a></p>
</li>
<li>
<p>Mita, T., &amp; <strong>Jombart, T.</strong> (2015). Patterns and dynamics of genetic
diversity in Plasmodium falciparum: What past human migrations tell
us about malaria. <i>Parasitology International</i>, <i>64</i>(3),
238-243. doi:<a href="http://doi.org/10.1016/j.parint.2014.09.007">10.1016/j.parint.2014.09.007</a></p>
</li>
<li>
<p>Devillard, S., <strong>Jombart, T.</strong>, Leger, F., Pontier, D., Say, L., &amp;
Ruette, S. (2014). How reliable are morphological and anatomical
characters to distinguish European wildcats, domestic cats and their
hybrids in France?. <i>Journal of Zoological Systematics and
Evolutionary Research</i>, <i>52</i>(2), 154-162.
doi:<a href="http://doi.org/10.1111/jzs.12049">10.1111/jzs.12049</a></p>
</li>
<li>
<p><strong>Jombart, T.</strong>, Aanensen, D. M., Baguelin, M., Birrell, P., Cauchemez,
S., Camacho, A., &hellip; Ferguso, N. (2014). OutbreakTools: A new
platform for disease outbreak analysis using the R software.
<i>EPIDEMICS</i>, <i>7</i>, 28-34. doi:<a href="http://doi.org/10.1016/j.epidem.2014.04.003">10.1016/j.epidem.2014.04.003</a></p>
</li>
<li>
<p><strong>Jombart, T.</strong>, Cori, A., Didelot, X., Cauchemez, S., Fraser, C., &amp;
Ferguson, N. (2014). Bayesian Reconstruction of Disease Outbreaks by
Combining Epidemiologic and Genomic Data. <i>PLOS COMPUTATIONAL
BIOLOGY</i>, <i>10</i>(1), 14 pages. doi:<a href="http://doi.org/10.1371/journal.pcbi.1003457">10.1371/journal.pcbi.1003457</a></p>
</li>
<li>
<p>Tanabe, K., <strong>Jombart, T.</strong>, Horibe, S., Palacpac, N. M. Q., Honma, H.,
Tachibana, S. -I., &hellip; Mita, T. (2013). Plasmodium falciparum
mitochondrial genetic diversity exhibits isolation-by-distance
patterns supporting a sub-Saharan African origin. <i>MITOCHONDRION</i>,
<i>13</i>(6), 630-636. doi:<a href="http://doi.org/10.1016/j.mito.2013.08.008">10.1016/j.mito.2013.08.008</a></p>
</li>
<li>
<p>Chadeau-Hyam, M., Campanella, G., <strong>Jombart, T.</strong>, Bottolo, L.,
Portengen, L., Vineis, P., &hellip; Vermeulen, R. C. H. (2013).
Deciphering the complex: Methodological overview of statistical
models to derive OMICS-based biomarkers. <i>ENVIRONMENTAL AND
MOLECULAR MUTAGENESIS</i>, <i>54</i>(7), 542-557.
doi:<a href="http://doi.org/10.1002/em.21797">10.1002/em.21797</a></p>
</li>
<li>
<p>Huyse, T., Van den Broeck, F., <strong>Jombart, T.</strong>, Webster, B. L., Diaw, O.,
Volckaert, F. A. M., &hellip; Polman, K. (2013). Regular treatments of
praziquantel do not impact on the genetic make-up of Schistosoma
mansoni in Northern Senegal. <i>INFECTION GENETICS AND EVOLUTION</i>,
<i>18</i>, 100-105. doi:<a href="http://doi.org/10.1016/j.meegid.2013.05.007">10.1016/j.meegid.2013.05.007</a></p>
</li>
<li>
<p>Tanabe, K., Mita, T., Palacpac, N. M. Q., Arisue, N., Tougan, T.,
Kawai, S., &hellip; Horii, T. (2013). Within-population genetic diversity
of Plasmodium falciparum vaccine candidate antigens reveals
geographic distance from a Central sub-Saharan African origin.
<i>VACCINE</i>, <i>31</i>(9), 1334-1339.
doi:<a href="http://doi.org/10.1016/j.vaccine.2012.12.039">10.1016/j.vaccine.2012.12.039</a></p>
</li>
<li>
<p>Cui, Y., Yu, C., Yan, Y., Li, D., Li, Y., <strong>Jombart, T.</strong>, &hellip; Yang, R.
(2013). Historical variations in mutation rate in an epidemic
pathogen, Yersinia pestis. <i>Proceedings of the National Academy of
Sciences of the United States of America</i>, <i>110</i>(2), 577-582.
doi:<a href="http://doi.org/10.1073/pnas.1205750110">10.1073/pnas.1205750110</a></p>
</li>
<li>
<p>Muenkemueller, T., Lavergne, S., Bzeznik, B., Dray, S., <strong>Jombart, T.</strong>,
Schiffers, K., &amp; Thuiller, W. (2012). How to measure and test
phylogenetic signal. <i>Methods in Ecology and Evolution</i>, <i>3</i>(4),
743-756. doi:<a href="http://doi.org/10.1111/j.2041-210X.2012.00196.x">10.1111/j.2041-210X.2012.00196.x</a></p>
</li>
<li>
<p>Dray, S., Pelissier, R., Couteron, P., Fortin, M. -J., Legendre, P.,
Peres-Neto, P. R., &hellip; Wagner, H. H. (2012). Community ecology in the
age of multivariate multiscale spatial analysis. <i>ECOLOGICAL
MONOGRAPHS</i>, <i>82</i>(3), 257-275. doi:<a href="http://doi.org/10.1890/11-1183.1">10.1890/11-1183.1</a></p>
</li>
<li>
<p>Yalcindag, E., Elguero, E., Arnathau, C., Durand, P., Akiana, J.,
Anderson, T. J., &hellip; Prugnolle, F. (2012). Multiple independent
introductions of Plasmodium falciparum in South America. <i>Proceedings
of the National Academy of Sciences of the United States of America</i>,
<i>109</i>(2), 511-516. doi:<a href="http://doi.org/10.1073/pnas.1119058109">10.1073/pnas.1119058109</a></p>
</li>
<li>
<p>Dray, S., &amp; <strong>Jombart, T.</strong> (2011). revisiting guerry&rsquo;s data:
introducing spatial constraints in multivariate analysis. <i>Snnals
of Applied Statistics</i>, <i>5</i>(4), 2278-2299.
doi:<a href="http://doi.org/10.1214/10-AOAS356">10.1214/10-AOAS356</a></p>
</li>
<li>
<p>Rasmussen, M., Guo, X., Wang, Y., Lohmueller, K. E., Rasmussen, S.,
Albrechtsen, A., &hellip; Willerslev, E. (2011). An Aboriginal Australian
Genome Reveals Separate Human Dispersals into Asia. <i>SCIENCE</i>,
<i>334</i>(6052), 94-98. doi:<a href="http://doi.org/10.1126/science.1211177">10.1126/science.1211177</a></p>
</li>
<li>
<p><strong>Jombart, T.</strong>, &amp; Ahmed, I. (2011). adegenet 1.3-1: new tools for
the analysis of genome-wide SNP data. <i>BIOINFORMATICS</i>, <i>27</i>(21),
3070-3071. doi:<a href="http://doi.org/10.1093/bioinformatics/btr521">10.1093/bioinformatics/btr521</a></p>
</li>
<li>
<p>Morelli, G., Song, Y., Mazzoni, C. J., Eppinger, M., Roumagnac, P.,
Wagner, D. M., &hellip; Achtman, M. (2010). Yersinia pestis genome
sequencing identifies patterns of global phylogenetic diversity.
<i>NATURE GENETICS</i>, <i>42</i>(12), 1140-+. doi:<a href="http://doi.org/10.1038/ng.705">10.1038/ng.705</a></p>
</li>
<li>
<p><strong>Jombart, T.</strong>, Devillard, S., &amp; Balloux, F. (2010). Discriminant
analysis of principal components: a new method for the analysis of
genetically structured populations. <i>BMC GENETICS</i>, <i>11</i>,
15 pages. doi:<a href="http://doi.org/10.1186/1471-2156-11-94">10.1186/1471-2156-11-94</a></p>
</li>
<li>
<p>Tanabe, K., Mita, T., <strong>Jombart, T.</strong>, Eriksson, A., Horibe, S.,
Palacpac, N., &hellip; Balloux, F. (2010). Plasmodium falciparum
Accompanied the Human Expansion out of Africa. <i>CURRENT BIOLOGY</i>,
<i>20</i>(14), 1283-1289. doi:<a href="http://doi.org/10.1016/j.cub.2010.05.053">10.1016/j.cub.2010.05.053</a></p>
</li>
<li>
<p><strong>Jombart, T.</strong>, Eggo, R. M., Dodd, P. J., &amp; Balloux, F. (2011).
Reconstructing disease outbreaks from genetic data: a graph approach.
<i>HEREDITY</i>, <i>106</i>(2), 383-390. doi:<a href="http://doi.org/10.1038/hdy.2010.78">10.1038/hdy.2010.78</a></p>
</li>
<li>
<p><strong>Jombart, T.</strong>, Balloux, F., &amp; Dray, S. (2010). adephylo: new tools
for investigating the phylogenetic signal in biological traits.
<i>BIOINFORMATICS</i>, <i>26</i>(15), 1907-1909.
doi:<a href="http://doi.org/10.1093/bioinformatics/btq292">10.1093/bioinformatics/btq292</a></p>
</li>
<li>
<p><strong>Jombart, T.</strong>, Pavoine, S., Devillard, S., &amp; Pontier, D. (2010).
Putting phylogeny into the analysis of biological traits: A
methodological approach. <i>JOURNAL OF THEORETICAL BIOLOGY</i>,
<i>264</i>(3), 693-701. doi:<a href="http://doi.org/10.1016/j.jtbi.2010.03.038">10.1016/j.jtbi.2010.03.038</a></p>
</li>
<li>
<p>Nuebel, U., Dordel, J., Kurt, K., Strommenger, B., Westh, H., Shukla,
S. K., &hellip; Witte, W. (2010). A Timescale for Evolution, Population
Expansion, and Spatial Spread of an Emerging Clone of
Methicillin-Resistant Staphylococcus aureus. <i>PLOS PATHOGENS</i>,
<i>6</i>(4), 12 pages. doi:<a href="http://doi.org/10.1371/journal.ppat.1000855">10.1371/journal.ppat.1000855</a></p>
</li>
<li>
<p>Balloux, F., Handley, L. -J. L., <strong>Jombart, T.</strong>, Liu, H., &amp; Manica,
A. (2009). Climate shaped the worldwide distribution of human
mitochondrial DNA sequence variation. <i>PROCEEDINGS OF THE ROYAL
SOCIETY B-BIOLOGICAL SCIENCES</i>, <i>276</i>(1672), 3447-3455.
doi:<a href="http://doi.org/10.1098/rspb.2009.0752">10.1098/rspb.2009.0752</a></p>
</li>
<li>
<p>Fraser, C., Donnelly, C. A., Cauchemez, S., Hanage, W. P., Van
Kerkhove, M. D., Hollingsworth, T. D., &hellip; Ferguson, N. M. (2009).
Influenza: Making Privileged Data Public Response. <i>SCIENCE</i>,
<i>325</i>(5944), 1072-1073. doi:<a href="http://doi.org/10.1126/science.325_1072b">10.1126/science.325_1072b</a></p>
</li>
<li>
<p>Fraser, C., Donnelly, C. A., Cauchemez, S., Hanage, W. P., Van
Kerkhove, M. D., Hollingsworth, T. D., &hellip; Roth, C. (2009). Pandemic
Potential of a Strain of Influenza A (H1N1): Early Findings. <i>SCIENCE</i>,
<i>324</i>(5934), 1557-1561. doi:<a href="http://doi.org/10.1126/science.1176062">10.1126/science.1176062</a></p>
</li>
<li>
<p><strong>Jombart, T.</strong>, Pontier, D., &amp; Dufour, A. -B. (2009). Genetic
markers in the playground of multivariate analysis. <i>HEREDITY</i>,
<i>102</i>(4), 330-341. doi:<a href="http://doi.org/10.1038/hdy.2008.130">10.1038/hdy.2008.130</a></p>
</li>
<li>
<p><strong>Jombart, T.</strong>, Dray, S., &amp; Dufour, A. -B. (2009). Finding essential
scales of spatial variation in ecological data: a multivariate
approach. <i>ECOGRAPHY</i>, <i>32</i>(1), 161-168.
doi:<a href="http://doi.org/10.1111/j.1600-0587.2008.05567.x">10.1111/j.1600-0587.2008.05567.x</a></p>
</li>
<li>
<p><strong>Jombart, T.</strong>, Eggo, R. M., Dodd, P., &amp; Balloux, F. (2009).
Spatiotemporal dynamics in the early stages of the 2009 A/H1N1
influenza pandemic.. <i>PLoS Curr</i>, <i>1</i>, RRN1026.</p>
</li>
<li>
<p>Devillard, S., <strong>Jombart, T.</strong>, &amp; Pontier, D. (2009). Revealing
cryptic genetic structuring in an urban population of stray cats
(Felis silvestris catus). <i>MAMMALIAN BIOLOGY</i>, <i>74</i>(1),
59-71. doi:<a href="http://doi.org/10.1016/j.mambio.2008.01.001">10.1016/j.mambio.2008.01.001</a></p>
</li>
<li>
<p>Christophe, F., Christl, A. D., Cauchemez, S., Hanage, W. P., Van
Kerkhove, M. D., Hollingsworth, T. D., &hellip; Ferguson, N. M. (2009).
Response. <i>Science</i>, <i>325</i>(5944), 1072-1073.</p>
</li>
<li>
<p><strong>Jombart, T.</strong>, Devillard, S., Dufour, A. -B., &amp; Pontier, D. (2008).
Revealing cryptic spatial patterns in genetic variability by a new
multivariate method. <i>HEREDITY</i>, <i>101</i>(1), 92-103.
doi:<a href="http://doi.org/10.1038/hdy.2008.34">10.1038/hdy.2008.34</a></p>
</li>
<li>
<p><strong>Jombart, T.</strong> (2008). adegenet: a R package for the multivariate
analysis of genetic markers. <i>BIOINFORMATICS</i>, <i>24</i>(11),
1403-1405. doi:<a href="http://doi.org/10.1093/bioinformatics/btn129">10.1093/bioinformatics/btn129</a></p>
</li>
<li>
<p>Laloe, D., <strong>Jombart, T.</strong>, Dufour, A. -B., &amp; Moazami-Goudarzi, K.
(2007). Consensus genetic structuring and typological value of
markers using multiple co-inertia analysis. <i>GENETICS SELECTION
EVOLUTION</i>, <i>39</i>(5), 545-567. doi:<a href="http://doi.org/10.1051/gse:2007021">10.1051/gse:2007021</a></p>
</li>
</ul>
</ul>
<br>
<br>
<br>

			</article>
			


		</div>
	</div>	

</main>

<footer id="footer">
  <div class="container">
    <div class="row">
      
      <div class="col-md-3 widget">
	<h3 class="widget-title">Contact</h3>
	<div class="widget-body">
	  <p><font color="white">Thibaut Jombart</font><br>
	    <br>
	    
	    <a href="mailto:thibautjombart@gmail.com">thibautjombart@gmail.com</a>
	  </p>
	</div>
      </div>
      

      
      <div class="col-md-3 widget">
	<h3 class="widget-title">Follow me</h3>
	<div class="widget-body">
	  <p class="follow-me-icons">
            
            
            <a href="mailto:thibautjombart@gmail.com" target="_blank"><i class="fa fa-envelope fa-2"></i></a>
            
            
            
            <a href="https://twitter.com/TeebzR" target="_blank"><i class="fa fa-twitter-square fa-2"></i></a>
            
            
            
            <a href="https://github.com/thibautjombart" target="_blank"><i class="fa fa-github fa-2"></i></a>
            
            
            
            <a href="https://www.facebook.com/thibaut.jombart" target="_blank"><i class="fa fa-facebook-square fa-2"></i></a>
            
            
	  </p>
	</div>
      </div>
      

      

      


      
      
      <div class="col-md-3 widget" >
	<h3 class="widget-title">Legal stuff</h3>
	<div class="widget-body">
	  
	  <p>
	    Copyright &copy; 2017, Thibaut Jombart<br>
	    Initial Hugo template: <a href="http://www.gettemplate.com" rel="designer">Initio by GetTemplate</a><br>
	    Additional design: Thibaut Jombart
            Powered by: <a href="https://gohugo.io/" rel="poweredby">Hugo</a> and <a href="https://bookdown.org/yihui/blogdown/">blogdown</a>.
          </p>
	</div>
      </div>

      

      
      
    </div> 
  </div>
</footer>





























<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
<script src="https://netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>
<script src="https://thibautjombart.netlify.app/js/template.js"></script>
<script id="dsq-count-scr" src="///count.js" async></script>
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', '', 'auto');
  ga('send', 'pageview');
</script>



<script src="//yihui.name/js/math-code.js"></script>
<script async
	src="//cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML">
</script>

</body>
</html>

